src/examples/regression_test.json

```diff
@@ -164,5 +164,18 @@
       "data": "",
       "hash": ""
     }
+  },
+  "Bitwise Primitive": {
+    "digest": "Bitwise Primitive",
+    "methods": {
+      "rot": {
+        "rows": 17,
```

Comment:
Where did this 17 come from? I am guessing this uses at least 1 row for Rot64, another 3 for RangeCheck0 (meaning excess, shifted, and the word), and a constant row with value 0 to check some cells were zero in the RangeCheck0's. But what about the rest?

Reply:
Oh, is this the BitwiseMock thing? I am seeing now that one produced 4 rotations, so maybe that's where 4*4+1 came from?